index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="flex-container header-items">
+    <a href="#" id="header-welcome-link">Welcome</a>
+    <img src="images/union.png" alt="nav button">
+  </header>
+  <section class="flex-container bg-img">
+    <div>
+      <p class="headline-paragraphs" id="p1">Hey there. I’m Mateo<br><span id="p3">I'm a software developer</span></p> 
```

Comment:
replace the p tag with h1 as it's the appropriate tag

Reply:
Done :heavy_check_mark::muscle: